README.md

```diff
@@ -2,6 +2,19 @@
 
 A standalone client library for interacting with the filecoin storage network
 
+## Usage
+Building the CLI:
+```bash
+make
+cd filc
+go build
+```
+
+Using it as a golang library:
+```bash
+go get github.com/application-research/filclient@latest
```

Comment:
we should probably add some info about the filecoin-ffi setup process

```suggestion
go get github.com/application-research/filclient@latest

# Add filecoin-ffi as an extern dependency
git clone https://github.com/filecoin-project/filecoin-ffi extern/filecoin-ffi
cd extern/filecoin-ffi
git checkout [filclient's dependency version]
make
cd ../..
go mod edit -replace github.com/filecoin-project/filecoin-ffi=./extern/filecoin-ffi
```

this process is really terrible and i wish we could do something about it